### PR TITLE
Resumable harvests

### PR DESCRIPTION
**Pull request by Reinkaos**

### CODE REVIEW

**Review by invke — APPROVED** [review]
Nice additions 👍

**Comment by Reinkaos** [comment]
Your force push upset me haha.

**Comment by paul-mesnilgrente** [comment]
Ah true sorry ^^

**Review by motizuki — APPROVED** [review]

File: app/models/harvest_job.rb [hunk 1]
```diff
@@ -53,6 +53,16 @@ def records
     options[:limit] = limit.to_i if limit.to_i.positive?
     options[:from] = parser.last_harvested_at if incremental? && parser.last_harvested_at
 
+    # pass details that are needed for resuming the job ...
+    options[:job] = self
+
+    if self.states.any?
+      options[:page]    = self.states.last.page
```
Comment (EdwinRozario) [inline app/models/harvest_job.rb:60]: Check if you need to call `self` here.

File: config/application.rb [hunk 2]
```diff
@@ -39,6 +39,13 @@ class Application < Rails::Application
     # custom global configs goes here
     config.eager_load_paths += %W[#{Rails.root}/lib]
     config.time_zone = 'Wellington'
+
+    config.after_initialize do
```
Comment (EdwinRozario) [inline config/application.rb:43]: Dont need this?

**Review by EdwinRozario — APPROVED** [review]

**Pull request merged**